tests/message_interface.rs

```diff
@@ -152,7 +152,8 @@ async fn build_and_post_block() {
 //                 {
 //                     "type":5,
 //                     "tag":"0x484f524e4554205370616d6d6572",
-//                     "data":"0x494f5441202d2041206e6577206461776e0a436f756e743a203030323330330a54696d657374616d703a20323032322d30342d32375431383a35343a30395a0a54697073656c656374696f6e3a203832c2b573"
+//
+// "data":"0x494f5441202d2041206e6577206461776e0a436f756e743a203030323330330a54696d657374616d703a20323032322d30342d32375431383a35343a30395a0a54697073656c656374696f6e3a203832c2b573"
```

Comment:
:carlos: ?

Reply:
I also wondered about this, but that's what `cargo +nightly fmt` is doing

Reply:
And in your PR it's failing because of this :trollface: 
https://github.com/iotaledger/iota.rs/actions/runs/3117688079/jobs/5056506179

Reply:
🙈